packages/react-core/src/components/Tile/Tile.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Tile/tile';
+import { css } from '@patternfly/react-styles';
+
+export interface TileProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the banner */
+  children?: React.ReactNode;
+  /** Additional classes added to the banner */
+  className?: string;
+  /** Title of the tile */
+  title: string;
+  /** Icon in the tile title */
+  icon?: React.ReactNode;
+  /** Flag indicating if the tile is selected */
+  isSelected?: boolean;
+  /** Flag indicating if the tile is disabled */
+  isDisabled?: boolean;
+  /** Flag indicating if the tile icon is stacked */
+  isStacked?: boolean;
+  /** Flag indicating if the stacked tile icon is large */
+  isLargeIcon?: boolean;
```

Comment:
I would rename this to `isDisplayLarge`. This variation could introduce other styles that don't apply to the icon. Also this is a new "display" concept we've added for more compact (application) vs larger (website/marketing) designs, which is why it would be `isDisplayLarge` instead of something like `isLarge`, which could do something different, like change the width or height or something like that and be different in concept to what `isDisplayLarge` does.

Reply:
Also that matches the core class, so we will avoid name conflicts in the future since these match.